webcompat/form.py

```diff
@@ -154,6 +159,22 @@ def normalize_url(url):
     return url
 
 
+def normalize_metadata(metadata_value):
+    """Normalize the metadata received from the form."""
+    # Removing closing comments.
+    if metadata_value is None:
+        return None
+    if '-->' in metadata_value:
+        metadata_value = metadata_value.replace('-->', '')
+        metadata_value = normalize_metadata(metadata_value)
+    # Let's avoid html tags in
+    if ('<' or '>') in metadata_value and '-->' not in metadata_value:
```

Comment:
At this point in the algorithm, when will we still have `'-->'` in `metadata_value`? Wouldn't it have been removed just above?

Reply:
yup. Caught in the recursive test. There is no more `-->`